src/sass/base/_images.scss

```diff
@@ -0,0 +1,11 @@
+.image-person-placeholder {
```

Comment:
Hi Rob, 
Thanks for suggesting these changes, 
I'm new to sass as well. I have made the changes and will check them in my next commit.